### PR TITLE
feedback handling to submission

### PR DESCRIPTION
**Pull request by Ralii**

### CODE REVIEW

File: src/actions/submissionsActions.jsx [hunk 1]
```diff
@@ -56,6 +68,22 @@ export const submitExercise = (exerciseId, feedbackAsked, fileContent, fileType)
       });
   };
 
+export const patchExercise = (submissionId, feedback) =>
+    (dispatch, getState) => {
+      const { id: userId, token } = getState().auth;
+      const axios = createAxios(token);
+      axios.put(`/submissions/${submissionId}`, {
+        feedback,
+        user_id: userId,
```
Comment (nfrolov) [inline src/actions/submissionsActions.jsx]: Remove user_id

File: src/components/SubmissionsSingleView.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    componentDidMount() {
+      if (this.props.submission.feedback !== null) {
+        this.refs.textarea.value = this.props.submission.feedback;
+      }
+    }
+
+    onSubmit(e) {
+      e.preventDefault();
+      console.log('PATCHÖÖÖ');
```
Comment (markolaine) [inline src/components/SubmissionsSingleView.jsx]: remove console.log

File: src/actions/submissionsActions.jsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ export const SUBMISSIONS_ADD_RESET = 'SUBMISSIONS_ADD_RESET';
 export const SUBMISSIONS_REQUEST = 'SUBMISSIONS_REQUEST';
 export const SUBMISSIONS_SUCCESS = 'SUBMISSIONS_SUCCESS';
 export const SUBMISSIONS_FAIL = 'SUBMISSIONS_FAIL';
+export const SUBMISSIONS_SUBMIT_FEEDBACK_SUCCESS = 'SUBMISSIONS_SUBMIT_FEEDBACK_SUCESS';
```
Comment (markolaine) [inline src/actions/submissionsActions.jsx]: sucess

File: src/components/RegistrationPage.jsx [hunk 4]
```diff
@@ -35,7 +35,7 @@ export default class RegistrationPage extends Component {
   renderForm() {
     const { inProgress, errorMessages } = this.props.reg;
     return (
-      <RegistrationForm loading={inProgress} errors={errorMessages} onSubmit={this.onSubmit} />
+      <RegistrationForm loading={inProgress} errors={errorMessages} onSubmit={this.onSubmit()} />
```
Comment (nfrolov) [inline src/components/RegistrationPage.jsx]: Remove

File: src/actions/submissionsActions.jsx (hunk 1 above)
Comment (nfrolov) [inline src/actions/submissionsActions.jsx]: Update done flag as well

File: src/components/SubmissionsSingleView.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                  </div>
+                  <div className="ui segment">
+                    <p>{exercise.text}</p>
+                  </div>
+                </div>
+              </div>
+              <div className="ten wide column">
+                <div className="ui segments">
+                  <div className="ui segment">
+                    <div className="ui form">
```
Comment (nfrolov) [inline src/components/SubmissionsSingleView.jsx]: Wrong nesting

**Pull request merged**